.gitmodules

```diff
@@ -0,0 +1,5 @@
+[submodule "kubernetes"]
+	path = vendor/k8s.io/kubernetes
+	url = https://github.com/openshift/kubernetes.git
+  branch = openshift-3.7
```

Comment:
release-1.7.0

Reply:
Not saying you are wrong, but openshift-specific branch naming might help reduce confusion for new contributors.